frontend/src/components/MediaGallery.tsx/index.scss

```diff
@@ -0,0 +1,34 @@
+.media-gallery:has(:nth-child(1)) {
```

Comment:
here I thought that using `:has` to check the number of children and change the grid accordingly was a cool solution, but this becomes the only code (except from the dynamic statuses contents) that is not using tailwind, so I am a bit torn on whether I should just change this to use tailwind.

The issue is that tailwind doesn't provide functional pseudo-classes (see this [tweet](https://twitter.com/adactio/status/1404467229857665027?lang=en)) so if I were to use tailwind I would have to have code that checks the number of medias and apply different classes accordingly.
I would also have to take into consideration the children and so something analogous for them...

I feel like that code would end up being a bit messy... but I can give it a shot if we think that would be the right direction?

any thoughts @jculvey @petebacondarwin ?

Reply:
I would stick with this class you've crafted if it works.